dags/bqetl_adjust_derived.py

```diff
@@ -0,0 +1,56 @@
+# Generated via https://github.com/mozilla/bigquery-etl/blob/main/bigquery_etl/query_scheduling/generate_airflow_dags.py
+
+from airflow import DAG
+from airflow.sensors.external_task import ExternalTaskMarker
+from airflow.sensors.external_task import ExternalTaskSensor
+from airflow.utils.task_group import TaskGroup
+import datetime
+from utils.constants import ALLOWED_STATES, FAILED_STATES
+from utils.gcp import bigquery_etl_query, gke_command, bigquery_dq_check
+
+docs = """
+### bqetl_adjust_derived
+
+Built from bigquery-etl repo, [`dags/bqetl_adjust_derived.py`](https://github.com/mozilla/bigquery-etl/blob/main/dags/bqetl_adjust_derived.py)
+
+#### Description
+
+Derived tables built on Adjust data downloaded from https://api.adjust.com/kpis/v1/<app_token>. Using mhirose's API token
+#### Owner
+
+mhirose@mozilla.com
+"""
+
+
+default_args = {
+    "owner": "mhirose@mozilla.com",
+    "start_date": datetime.datetime(2023, 7, 6, 0, 0),
+    "end_date": None,
+    "email": ["telemetry-alerts@mozilla.com", "mhirose@mozilla.com"],
+    "depends_on_past": False,
+    "retry_delay": datetime.timedelta(seconds=1800),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+}
+
+tags = ["impact/tier_2", "repo/bigquery-etl"]
```

Comment:
Just to check, is this data not tier 1? --which would imply data and the highest level of support.

Reply:
The data isn't super high priority, so that is why I made it tier 2. The original table/DAG is tier 2